sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/LogicalRelation.scala

```diff
@@ -102,3 +109,18 @@ object LogicalRelation {
     LogicalRelation(relation, toAttributes(schema), Some(table), false)
   }
 }
+
+/**
+ * Extract the [[BaseRelation]] and [[CatalogTable]] from [[LogicalRelation]]. This also provides
+ * [[LogicalRelation]] itself, to cover the case where other params are needed (but not required
```

Comment:
shall we update this comment?

Reply:
Thanks for catching this, forgot to change.